bin/tink.js

```diff
@@ -6,6 +6,10 @@ const CMDS = new Map([
   ['sh', require('../lib/commands/shell.js')],
   ['shell', require('../lib/commands/shell.js')],
   ['org', require('../lib/commands/org.jsx')],
+  ['s', require('../lib/commands/search.jsx')],
+  ['se', require('../lib/commands/search.jsx')],
+  ['find', require('../lib/commands/search.jsx')],
+  ['search', require('../lib/commands/search.jsx')],
```

Comment:
I don't think we need this duplication anymore since I made some changes to the way commands get loaded. Wanna try without?